Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/GstVerification.hs

```diff
@@ -201,9 +185,115 @@ verifyGstin verifyBy mbMerchant (personId, _, merchantOpCityId) req adminApprova
+    && (output.gstinStatus /= Just "Active")
+    then do
+      case serviceName of
+        VT.Idfy -> IVQuery.updateExtractValidationStatus Domain.Failed verificationReq.requestId
+        _ -> throwError $ InternalError ("Unknown Service provider webhook encopuntered in onVerifyDL. Name of provider : " <> show serviceName)
+      pure Ack
+    else do
+      onVerifyGstHandler person verificationReq.documentImageId1 verificationReq.documentImageId2 output
+      pure Ack
+
+onVerifyGstHandler :: Person.Person -> Id Image.Image -> Maybe (Id Image.Image) -> VT.GstVerificationResponse -> Flow ()
+onVerifyGstHandler person imageId1 imageId2 output = do
+  mEncryptedGstinNumber <- encrypt `mapM` output.gstin
+  let isValidGst = output.gstinStatus == Just "Active"
+  DGQuery.updateStrictVerificationStatusByDriverId (Just isValidGst) person.id
+  case person.role of
+    role | DCommon.checkFleetOwnerRole role -> do
+      QFOI.updateGstImage mEncryptedGstinNumber (Just imageId1.getId) person.id
+    _ -> pure ()
+  (image1, image2) <- uncurry (liftA2 (,)) $ both (maybe (return Nothing) ImageQuery.findById) (Just imageId1, imageId2)
+  when (((image1 >>= (.verificationStatus)) /= Just Documents.VALID) && ((image2 >>= (.verificationStatus)) /= Just Documents.VALID)) $
+    mapM_ (maybe (return ()) (ImageQuery.updateVerificationStatusAndFailureReason Documents.VALID (ImageNotValid "verificationStatus updated to VALID by dashboard."))) [Just imageId1, imageId2]
+  pure ()
+
+buildGstinCard :: Person.Person -> Maybe Text -> Maybe Text -> Maybe Text -> Maybe Bool -> Maybe Text -> Maybe Text -> Maybe Text -> Maybe Text -> Maybe Text -> Maybe Text -> Maybe DPan.VerifiedBy -> Id Image.Image -> Text -> Flow DGst.DriverGstin
+buildGstinCard person address constitution_of_business date_of_liability is_provisional legal_name trade_name type_of_registration valid_from valid_upto pan_number verifyBy image1 gstIn = do
+  gstinEnc <- encrypt gstIn
+  now <- getCurrentTime
+  uuid <- generateGUID
+  return $
+    DGst.DriverGstin
+      { documentImageId1 = image1,
+        driverId = person.id,
+        id = uuid,
+        verificationStatus = Documents.VALID,
+        merchantId = Just person.merchantId,
+        merchantOperatingCityId = Just person.merchantOperatingCityId,
+        createdAt = now,
+        address = address,
+        constitutionOfBusiness = constitution_of_business,
+        updatedAt = now,
+        documentImageId2 = Nothing,
+        dateOfLiability = date_of_liability >>= DVRC.parseDateTime,
+        driverName = Just person.firstName,
+        gstin = gstinEnc,
+        isProvisional = is_provisional,
+        panNumber = pan_number,
+        legalName = legal_name,
+        tradeName = trade_name,
+        typeOfRegistration = type_of_registration,
+        validFrom = valid_from >>= DVRC.parseDateTime,
+        validUpto = valid_upto >>= DVRC.parseDateTime,
+        verifiedBy = verifyBy,
+        isStrictlyVerified = Nothing
```

Comment:
Could you explain why this this new field `isStrictlyVerified` required, if it always set as `Nothing`?